src/logging.cc

```diff
@@ -1153,7 +1153,9 @@ static LogMessage::LogMessageData fatal_msg_data_shared;
 // LogMessageData object exists (in this case glog makes zero heap memory
 // allocations).
 static GLOG_THREAD_LOCAL_STORAGE bool thread_data_available = true;
-static GLOG_THREAD_LOCAL_STORAGE char thread_msg_data[sizeof(LogMessage::LogMessageData)];
+static GLOG_THREAD_LOCAL_STORAGE std::aligned_storage<
+    sizeof(LogMessage::LogMessageData),
+    alignof(LogMessage::LogMessageData)>::type thread_msg_data;
```

Comment:
`alignof` is a C++11 keyword, is it not? There should be a fallback for C++98/03.